yklua/README.md

```diff
@@ -0,0 +1,19 @@
+# YkLua benchmarks
+
+[YkLua](https://github.com/ykjit/yklua)
+
+## Benchmarks:
+
+- [fannkuchredux](https://benchmarksgame-team.pages.debian.net/benchmarksgame/program/fannkuchredux-lua-1.html)
+
+- [fasta](https://benchmarksgame-team.pages.debian.net/benchmarksgame/program/fasta-lua-2.html)
+
+- [binary-trees](https://benchmarksgame-team.pages.debian.net/benchmarksgame/program/binarytrees-lua-2.html)
+
+## Running 
+
+```shell
+$ YKD_SERIALISE_COMPILATION=1 /path/to/yklua/src/lua ./fannkuchredux.lua 3
```

Comment:
Let's drop the `YKD_SERIALISE_COMPILATION` bits. They're not something we'll want to do, in general.

Reply:
done